dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -736,10 +737,17 @@ fn cmd_blueprint_edit(
     let blueprint = sim.blueprint_lookup(blueprint_id)?;
     let creator = args.creator.as_deref().unwrap_or("reconfigurator-cli");
     let planning_input = sim.planning_input(blueprint)?;
+    let latest_collection = sim
+        .collections
+        .iter()
+        .min_by_key(|(_, c)| c.time_started)
```

Comment:
If we want the latest collection, should this be max_by_key?

Reply:
Good catch. What was I thinking? 🤦